src/plugins/unified_histogram/public/chart/chart.tsx

```diff
@@ -296,6 +347,21 @@ export function Chart({
                     </EuiFlexItem>
                   </>
                 )}
+                {canEditVisualizationOnTheFly && (
```

Comment:
Not a blocker for this PR but what can become unexpected for users is that custom chart configuration is not going to be saved with a saved search although it's done on the same page.

<img width="300" alt="Screenshot 2023-06-30 at 14 01 34" src="https://github.com/elastic/kibana/assets/1415710/30be7b5d-a960-4ad1-bf76-7aa531ad5b23">

Reply:
Yes we don't want this to be part of the saved search for now, but I get the point. I would wait for users' feedback here to see how they feel about it.